pep-0282.txt

```diff
@@ -624,7 +624,7 @@ References
        http://starship.python.net/crew/jbauer/creosote/
 
 .. [6] Vinay Sajip's logging module.
-       http://www.red-dove.com/python_logging.html
+       http://old.red-dove.com/python_logging.html
```

Comment:
```suggestion
       https://old.red-dove.com/python_logging.html
```

May as well upgrade it to TLS at the same time, since it indeed appears to work